product-service/src/handlers/catalogBatchProcess.ts

```diff
@@ -0,0 +1,58 @@
+import { SNS } from 'aws-sdk';
+import { PublishInput } from 'aws-sdk/clients/sns';
+import productProviderDynamoDB from '../dynamodb-providers/product.provider';
+import { HttpResponse } from '../helpers/http-response';
+import { isJson } from '../utils/isJson';
+
+const catalogBatchProcess = (event: any, context: any, cb: any) => {
```

Comment:
[minor]
You can use typings from 'aws-lambda' package for convenience (for event, context) objects

Reply:
[minor]
Also it's more widespreaded now to use async/await syntax for lambdas - no need to remember to call calback => you just return value from functions

`const catalogBatchProcess = async(event, context) => { ... }`